packages/core/addon/components/navi-visualizations/apex-gauge.js

```diff
@@ -0,0 +1,78 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import numeral from 'numeral';
+import layout from '../../templates/components/navi-visualizations/apex-gauge';
+
+export default class NaviVisualizationsApexGauge extends Component {
+  layout = layout;
+
+  /**
+   * @property {number} - the current value of the metric being displayed
+   */
+  @computed('model')
+  get data() {
+    const metric = this.model.firstObject.request.metrics['0'].metric;
+    return this.model?.firstObject?.response?.rows?.[0]?.[metric];
+  }
+
+  /**
+   * @property {array} - an array containing the progress percentage
+   */
+  @computed('options', 'data')
+  get series() {
+    const val =
+      100 *
+      ((this.data - Number(this.options.baselineValue)) /
+        (Number(this.options.goalValue) - Number(this.options.baselineValue)));
+    return [val];
+  }
+
+  /**
+   * @property {string} - the hex color of the gauge
+   */
+  @computed('series')
+  get gaugeColor() {
+    // thresholds:
+    // red < 75 <= yellow <= 85 < green
+    if (this.series[0] < 75) {
+      return '#f05050';
```

Comment:
Can we declare these color codes as consts and get rid of the comment?
`const RED = '#f05050';` etc.